server/src/games/Games.ClankCatacombs/ClankCatacombsScoringStrategy.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Text.Json;
+using BGS.ApplicationCore.Games.CalculateScore;
+
+namespace BGS.Games.ClankCatacombs;
+
+internal class ClankCatacombsScoringStrategy : IGameScoringStrategy
+{
+    public int Calculate(JsonElement scoringData)
```

Comment:
I think we need to add a condition when "ArtefactsScore" is 0, then the total score should be 0, because from the rules:
- The value of their Artifact (You can't score any points without one!)

also add tests